pages/tutorials/3.0/audio/sounds.md

```diff
@@ -190,3 +224,24 @@ void doSomething(sf::Music music)
 sf::Music music;
 doSomething(music); // ERROR (the function should take its argument by reference, not by value)
 ```
+
+`sf::Music` is however moveable.
+You can transfer ownership of a music object like this:
+
+```cpp
+class MyMusic
+{
+public:
+    MyMusic(sf::Music&& music) : m_music(std::move(music))
```

Comment:
Isn't that one too many moves?

Reply:
```
Testbed.cpp:6:34: error: call to implicitly-deleted copy constructor of 'sf::Music'
    6 |     MyMusic(sf::Music&& music) : m_music(music)
      |                                  ^       ~~~~~
```
The compiler wasn't happy unless I added two `std::move`s. Not entirely sure why.